utils/create_srg_export.py

```diff
@@ -465,16 +498,26 @@ def get_rule_json(json_path: str) -> dict:
 
 def check_paths(control_path: str, rule_json_path: str) -> None:
     control_full_path = pathlib.Path(control_path).absolute()
-    if not pathlib.Path.exists(control_full_path):
+    if not control_full_path.exists():
         sys.stderr.write(f"Unable to find control file {control_full_path}\n")
         exit(5)
     rule_json_full_path = pathlib.Path(rule_json_path).absolute()
-    if not os.path.exists(rule_json_full_path):
+    if not rule_json_full_path.exists():
         sys.stderr.write(f"Unable to find rule_dirs.json file {rule_json_full_path}\n")
         sys.stderr.write("Hint: run ./utils/rule_dir_json.py\n")
```

Comment:
Nit: Would it be interesting to already call the script instead of asking the used to do it? It could ask a confirmation, but already do that. It would also make sure the `rule_dirs.json` file is updated.

Reply:
So I briefly looked into this and it wouldn't be pure Python call, at least not in the current state of `rule_dir_json.py`. Also, the behavior in the current script follows our other tools use the file.

Reply:
Ok, better keep it that way then. Thanks for this feedback.